src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```diff
@@ -531,12 +531,8 @@ Side column size (first and second column from left)
 */
 
 .col-md-3 {
-  flex: none;
-  max-width: fit-content;
-}
-
-.col-xl-7 {
-  max-width: fit-content;
+    flex: 0 0 20%;
+    max-width: 20%;
```

Comment:
Let's keep an eye on it because in the past having the "size" of a panel based on percentage was the cause of some issues.

Reply:
Yeah... PyGeometry suffered of this issue with the percentage... Let's see